### ISSUE TITLE
Conditions involving XMM are always true

### ISSUE THREAD

**Issue by TypicalDarkness** [x64dbg/x64dbg#2314]
Version: Feb 23 2020 16:30:59
OS: Win 10.0.18362 64 bit

Breakpoints with conditions that involve XMM registers are always triggered, even if the condition is false:
For example:
`XMM0==5` evemthough XMM0 is not 5
`XMM0==5 && false`

**Comment by mrexodia** [comment]
Yes, xmm is not a valid expression. You could write a plugin that exposes an expression function like xmm0.iszero() and try this way.

**Comment by TypicalDarkness** [comment]
May I ask why it's not a valid expression? Is it because of a technical limitation? Sorry, I don't know much about CPUs

**Comment by mrexodia** [comment]
Inside x64dbg all expressions have the type equivalent to uintptr_t so it’s
currently not possible to use strings or things like xmm registers in them
because their value doesn’t fit in a uintptr_t.

**Comment by TypicalDarkness** [comment]
This wouldn't be a perfect solution, but would it be possible to split the registers into two halves, like xmm00 and xmm01.

Thanks for the quick response btw :)

**Comment by mrexodia** [comment]
Yeah it might be possible. Let's see.
